docs/orleans/streaming/streams-quick-start.md

```diff
@@ -1,27 +1,50 @@
 ---
 title: Orleans streaming quickstart
 description: Learn from the streaming quickstart in .NET Orleans.
-ms.date: 12/06/2022
+ms.date: 01/05/2023
 zone_pivot_groups: orleans-version
 ---
 
 # Orleans streaming quickstart
 
-This guide will show you a quick way to set up and use Orleans Streams.
-To learn more about the details of the streaming features, read other parts of this documentation.
+This guide will show you a quick way to set up and use Orleans Streams. To learn more about the details of the streaming features, read other parts of this documentation.
 
 ## Required configurations
 
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
```

Comment:
This is in other places in addition to this one, but MD044 is [Proper names should have the correct capitalization](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md044---proper-names-should-have-the-correct-capitalization). The rules for tabs are MD024/25. However, the config in the repo has them disabled, so I don't think you even need this.

Reply:
It's complaining that the zone pivot identifier, `orleans-` isn't capitalized as it's marked as a proper name.